buildenv/jenkins/jobs/infrastructure/copyrightCheck

```diff
@@ -38,7 +38,7 @@ timeout(time: 6, unit: 'HOURS') {
                                             noTags: false,
                                             reference: '/home/jenkins/openjdk_cache',
                                             shallow: false]],
-                            userRemoteConfigs: [[refspec: "+refs/pull/${ghprbPullId}/*:refs/remotes/origin/pr/${ghprbPullId}/*",
+                            userRemoteConfigs: [[refspec: "+refs/pull/${ghprbPullId}/*:refs/remotes/origin/pr/${ghprbPullId}/* +refs/heads/*:refs/remotes/origin/*",
```

Comment:
Do we really need them all?

Reply:
the issue that started this is that ibmruntimes/openjdk-jdk does not have a master branch. I suppose we could use the target variable. Let me update...